lib/util/http/bitbucket.ts

```diff
@@ -60,27 +70,13 @@ export class BitbucketHttp extends Http<BitbucketHttpOptions> {
   }
 }
 
-function getPageFromURL(url: string | undefined): string | null {
-  const resolvedURL = parseUrl(url);
-
-  if (is.nullOrUndefined(resolvedURL)) {
-    return null;
-  }
-
-  return resolvedURL.searchParams.get('page');
+function hasPagelen(url: URL): boolean {
+  return !is.nullOrUndefined(url.searchParams.get('pagelen'));
 }
 
-function getNextPagePath(path: string, nextPage: string): string | null {
-  const resolvedURL = parseUrl(resolveBaseUrl(baseUrl, path));
-
-  // istanbul ignore if
-  if (is.nullOrUndefined(resolvedURL)) {
-    return null;
-  }
-
-  resolvedURL.searchParams.set('page', nextPage);
-
-  return resolvedURL.toString();
+function addPagelenToPath(url: URL, pagelen: number): URL {
+  url.searchParams.set('pagelen', pagelen.toString());
```

Comment:
i think this should now inlined, it's only used once

Reply:
Agreed.  inlined